sass/directives/_carousels.scss

```diff
@@ -105,7 +105,10 @@
   font-size: $direction-arrow-size / 2;
   cursor: pointer;
   border: 1px solid $inactive-carousel-button-color;
-  padding: $base-padding / 2;
+  text-align: center;
+  line-height: 28px;
```

Comment:
will you please add a comment here that the line height is for vertical centering (could be "vertically centers arrows")? it will call this out just in case there's a better way (now or in the future) to vertically center this and if anything changes to throw the arrows off center it will be obvious that we need to update the line height